repos/system_upgrade/common/actors/initramfs/upgradeinitramfsgenerator/tests/unit_test_upgradeinitramfsgenerator.py

```diff
@@ -59,13 +59,23 @@ def gen_RUIP(packages):
     return RequiredUpgradeInitramPackages(packages=packages)
 
 
-def gen_UIT(modules, files):
-    if not isinstance(modules, list):
-        modules = [modules]
+def gen_UIT(dracut_modules, kernel_modules, files):
+    if not isinstance(dracut_modules, list):
```

Comment:
Nit: Maybe let's reduce code duplication here?
```
def _ensure_list(data):
    return data if isinstance(data, list) else [data]
```
All occurrences at L72-L74 will be using `_ensure_list`, like
```
dracut_modules = [DracutModule(name=i[0], module_path=i[1]) for i in _ensure_list(dracut_modules)]
```

Reply:
Yeah I do agree that a bit of cleaning would make it much more readable. Fixed!